magetab_validation.py

```diff
@@ -6,7 +6,6 @@
 
 import argparse
 import os
-import logging
```

Comment:
if you can, it is a good practice to simply ask for the exact elements of what you need, through the idiom `from x import y`. Then you don't load everything into memory and your programs load slightly faster and are leaner.

Reply:
specially for large packages (like numpy, pandas, etc) and if you are using only few methods.

Reply:
As I understood the importing mechanism, this doesn't make a difference for memory. But I'll read up on this some more.